lib/color-validate.spec.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+const { expect } = require('chai');
+const validColor = require('./color-validate.js');
+
+describe('Color validater', function () {
+  it('should check if given valid hex colors', function () {
```

Comment:
These are all data-driven tests. Could you use sazerac instead of adding your own loop code? In addition to being a little less code to maintain, it has the advantage of running all the examples, even if one of them fails.

Reply:
have removed these test and updated `make-badge.spec.js` test to use `sazerac`